app/src/main/res/layout/activity_contributors.xml

```diff
@@ -1,6 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <layout xmlns:android="http://schemas.android.com/apk/res/android"
         xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:tools="http://schemas.android.com/tools"
+        tools:context=".view.activity.ContributorsActivity"
```

Comment:
I have a question 🙋 
I know this `tools:context` is set when I create Activity by AndroidStudio file template.
But to be honest, I don't know what is the benefit to specify `tools:context`.